packages/ios_platform_images/ios/Classes/UIImageIosPlatformImages.swift

```diff
@@ -2,9 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#import <UIKit/UIKit.h>
-
-@interface UIImage (ios_platform_images)
+import Flutter
+import Foundation
+import SwiftUI
```

Comment:
why are we importing swiftui?

Reply:
i was testing another thing. Removed now.